scripts/build-single-manifests.sh

```diff
@@ -6,13 +6,23 @@ set -o pipefail
 
 REPO_ROOT=$(dirname ${BASH_SOURCE})/..
 
-cd $REPO_ROOT
+cd "${REPO_ROOT}"
 
-${REPO_ROOT}/bin/kustomize build config/base > deploy/single/all-in-one-dbless-legacy.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/postgres > deploy/single/all-in-one-postgres.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/enterprise > deploy/single/all-in-one-dbless-k4k8s-enterprise.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/enterprise-postgres > deploy/single/all-in-one-postgres-enterprise.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/multi-gw/oss > deploy/single/all-in-one-dbless.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/multi-gw/enterprise > deploy/single/all-in-one-dbless-enterprise.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/konnect/oss > deploy/single/all-in-one-dbless-konnect.yaml
-${REPO_ROOT}/bin/kustomize build config/variants/konnect/enterprise > deploy/single/all-in-one-dbless-konnect-enterprise.yaml
+# kustomize expests parameters:
+# Params:
+#   - $1 - Path to the configuration file
+#   - $2 - Path to resulting file
+kustomize(){
+    echo -e '# Generated by build-single-manifest.sh. DO NOT EDIT.\n' > "${2}"
```

Comment:
```suggestion
    echo -e '# Generated by build-single-manifest.sh. DO NOT EDIT.' > "${2}"
```

I think we better ditch the `\n` to prevent the empty line below the comment. WDYT?

Reply:
Firstly I had it without this empty line, but later I added it to indicate that comment belongs to the whole file, not the first K8s object defined in it

Reply:
Ah ok this makes sense

Reply:
And it's consistent with e.g. 
https://github.com/Kong/kubernetes-ingress-controller/blob/ea6e0e50eadc3ea13c4e41e90fa6d64135c6a5bb/pkg/clientset/typed/configuration/v1beta1/configuration_client.go#L17-L19
but for me, it can be either way...